dfusion_rust_core/src/models/order.rs

```diff
@@ -32,6 +34,27 @@ impl Serializable for u128 {
     }
 }
 
+impl From<Vec<u8>> for Order {
+    fn from (bytes: Vec<u8>) -> Self {
+        let mut bytes = bytes.clone();
```

Comment:
I think we should not need to clone here.

Reply:
Yep, there's no need for it now. 

It would haven been required for how I was going to implement it; by using the pop_from_log_data methods that mutate the vars. So this lines shadow the bytes and make it mutable, so we can extract the params of the orders.

The thing I found doing this is that we don't have a uint96 version of those pop methods...
Maybe is best just to take the slices from the Vec and getting the values in an immutable way without using those utils

Reply:
Even if you need this variable to be mutable, you don't need to clone. You could move bytes into the mutable version:

```
let mut bytes = bytes
```

This will consume the immutable version of bytes and make it available under the mutable variable.